client/blocks/post-item/index.jsx

```diff
@@ -182,12 +193,17 @@ export default connect(
 
 		const siteId = post.site_ID;
 
+		// Avoid rendering an external link while loading.
+		const externalPostLink = false === canCurrentUserEditPost( state, globalId );
```

Comment:
Super minor suggestion. Can we name this `isExternalPostLink`, `isPostLinkExternal`, or something similar? It's much easier to grok that it's a boolean that way. I expected that this contained the URL at first glance.

Reply:
+1, boolean variables should be prefixed with `is`.